spec/requests/api/v1/notifications_spec.rb

```diff
@@ -6,12 +6,21 @@
   let!(:admin_user) { create(:confirmed_user, admin: true) }
   let!(:logged_in_user) { create(:confirmed_user, :nebraska) }
   let!(:business) { create(:business, :nebraska_ldds, user: logged_in_user) }
-  let!(:approval) { create(:approval, num_children: 3, business: business) }
+  let!(:approval) { create(:approval, num_children: 3, business: business, expires_on: 3.days.after) }
+  let!(:early_approval) { create(:approval, num_children: 1, business: business, expires_on: 1.day.after) }
+  let!(:late_approval) { create(:approval, num_children: 1, business: business, expires_on: 29.days.after) }
   let!(:children) { approval.children }
+  let!(:early_child) { early_approval.children.first }
+  let!(:late_child) { late_approval.children.first }
   let!(:non_owner_child) { create(:necc_child) }
 
   before do
-    create(:notification, child: children.first, approval: approval)
+    approvals = Array.new(3) { |_i| approval }
```

Comment:
I'm a little confused about this configuration/setup, can you explain what your goal is here?